gno.land/cmd/gnoland/secrets_init.go

```diff
@@ -0,0 +1,141 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/gnolang/gno/tm2/pkg/commands"
+)
+
+type secretsInitCfg struct {
+	commonAllCfg
+}
+
+// newSecretsInitCmd creates the secrets init command
+func newSecretsInitCmd(io commands.IO) *commands.Command {
+	cfg := &secretsInitCfg{}
+
+	return commands.NewCommand(
+		commands.Metadata{
+			Name:       "init",
+			ShortUsage: "secrets init [flags] [<key>]",
+			ShortHelp:  "initializes required Gno secrets in a common directory",
+			LongHelp: fmt.Sprintf(
```

Comment:
idea: what if this command returned an error when the files already exist; and allows overwriting them with `--force` only?

the purpose would be to try to avoid accidental overwriting of important data through the `init` command.

Reply:
Added overwrite protection:

[cf5abb6](https://github.com/gnolang/gno/pull/1593/commits/cf5abb69186eceaadf3781cc624c0c75d557ce4e)